source/retryable-writes/tests/README.rst

```diff
@@ -9,8 +9,14 @@ Retryable Write Tests
 Introduction
 ============
 
-The YAML and JSON files in this directory tree are platform-independent tests
-that drivers can use to prove their conformance to the Retryable Writes spec.
+Tests in this directory are platform-independent tests that drivers can use to
+prove their conformance to the Retryable Writes specification.
+
+Tests in the ``unified`` directory are implemented in the
+`Unified Test Format <../../unified-test-format/unified-test-format.rst>`__
+and require schema version 1.0.
```

Comment:
I'd suggest removing "and require schema version 1.0", as this seems very likely to get out-of-date with a test change (e.g. needing to skip something for serverless). And it doesn't add much since the schema version for each test is already expressed within.

Reply:
I copied that from another spec which had it, but I agree with you.  

Done.